### PR TITLE
Mention label copying in the docs

### PR DESCRIPTION
**Pull request by pajakd**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:
This PR mentions in the documentation about the feature of label copying into the Workload object. 

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: site/content/en/docs/concepts/workload.md [hunk 1]
```diff
@@ -151,6 +151,10 @@ status:
 When a Workload deactivated by Sequential Admission with Ready Pods is re-activated, 
 the requeueState (`.status.requeueState`) will be reset to null.
 
+## Copying labels
+At Workload creation, Kueue has a functionality to copy labels into the Workload from the underlying Job or Pod objects. You can specify which labels should be copied by setting `labelKeysToCopy` field in the configuration API (under `Integrations`).
```
Comment (tenzen-y) [inline site/content/en/docs/concepts/workload.md:155]: Could we mention the default behavior something like this?

```
By default, Kueue copies all Job or Pod labels into the Workload.
```

**Review by tenzen-y — COMMENTED** [review]
Basically, lgtm.
Thank you!

File: site/content/en/docs/concepts/workload.md (hunk 1 above)
Comment (alculquicondor) [reply site/content/en/docs/concepts/workload.md:155]: Actually, it copies none :)
But yes, we should add the default behavior.

Comment (tenzen-y) [reply site/content/en/docs/concepts/workload.md:155]: Oops, I am misunderstanding the default behavior. 
Thanks!

**Comment by pajakd** [comment]
Explained the default, thanks for the comments/suggestions!

File: site/content/en/docs/concepts/workload.md (hunk 1 above)
Comment (pajakd) [reply site/content/en/docs/concepts/workload.md]: Added an explanation of the default

**Review by tenzen-y — COMMENTED** [review]
Thank you!
/lgtm
/approve

**Pull request merged**